Assets/Scripts/UI/SimulationInfoUIControllers/PatrollingInfoUIController.cs

```diff
@@ -87,12 +87,9 @@ protected override void AfterStart()
         }
 
 
-        public void Update()
+        public override void Update()
```

Comment:
Please remove update from the base class as it is just empty.

Reply:
Fixed

Reply:
Fixed